vassal-app/src/main/java/VASSAL/build/module/NotesWindow.java

```diff
@@ -62,6 +62,9 @@ public class NotesWindow extends AbstractToolbarItem
   public static final String BUTTON_TEXT = "buttonText"; //NON-NLS // non-standard legacy difference from AbstractToolbarItem
   public static final String DESCRIPTION = "description"; //NON-NLS
 
+  public static final int TAB_INDEX_SCEN = 0;
+  public static final int TAB_INDEX_PUBLIC = 1;
```

Comment:
Why make these public?

Reply:
Oh definitely no good reason. Was blindly copying what was right above haha

Reply:
(of course I predict several rounds of negotiation with style-enforcement-bot over the "right" way to make something non-public... mark my words)

Reply:
(which is why I tend to copy something already there, for better or worse)